tests/extended/TestAllIndividualPackages.cs

```diff
@@ -1,4 +1,5 @@
 using System.Text;
+using System.Xml;
```

Comment:
Unneeded.

Reply:
It was needed, but extended tests were not hit on local builds and PR builds.


https://github.com/dotnet/android-libraries/blob/dev/moljac/wsu-20250417/tests/allpackages/TestAllIndividualPackages.cs#L2